ambari-server/src/main/resources/stack-hooks/after-INSTALL/scripts/shared_initialization.py

```diff
@@ -84,7 +84,7 @@ def setup_config():
     XmlConfig("core-site.xml",
               conf_dir=params.hadoop_conf_dir,
               configurations=params.module_configs.get_property_value(params.module_name, 'core-site', ''),
-              configuration_attributes=params.execution_command._execution_command.__get_value("configurationAttributes/core-site"),
+              configuration_attributes=params.execution_command.get_value("configurationAttributes/core-site"),
```

Comment:
Let ModuleConfigs deal with configs and config attributes and add necessary methods to support retrieving config properties and config attributes.

The whole purpose of having the execution_command library is to hide the schema of command.json and having to call with "configurationAttributes/core-site" defeats the purpose. 

Example: 
```
class ModuleConfigs(object):
  """
  This class maps to "/configurations" block in command.json which includes configuration information of a service
  """

  def __init__(self, configs, config_attributes):
    self.__module_configs = configs
    self.__module_config_attributes = config_attributes    

  def get_attributes(self, module_name, config_type):
     return  config_attributes(config_type)

  def get_properties(self, module_name, config_type, property_names):
    return map(lambda property_name: self.get_property_value(module_name, config_type, property_name), property_names)

  def get_property_value(self, module_name, config_type, property_name, default=None):
    try:
      return self.__module_configs[config_type][property_name]
    except:
      return default
```